set.mm

```diff
@@ -393194,6 +393251,9 @@ Besides the usual way to represent graphs without edges (consisting of
   Other classes should not be used to represent graphs, because there could be
   a degenerated behavior of the vertex set and (indexed) edge functions, see
```

Comment:
degenerated --> degenerate (there are other instances below)

Reply:
I think both forms exist and are possible, but "degenerate" is maybe more appropriate (it is an inherent property of these cases, and not by an action applied to it). I will fix this.

Reply:
Done

Reply:
> it is an inherent property of these cases, and not [the result of] an action applied to it

exactly